scripts/stac/imagery/collection.py

```diff
@@ -130,6 +131,19 @@ def add_capture_area(self, polygons: list[BaseGeometry], target: str, artifact_t
         if StacExtensions.file.value not in self.stac["stac_extensions"]:
             self.stac["stac_extensions"].append(StacExtensions.file.value)
 
+    def add_capture_dates(self, source_directory: str) -> None:
+        capture_dates_content = read(os.path.join(source_directory, CAPTURE_DATES_FILE_NAME))
+        file_checksum = checksum.multihash_as_hex(capture_dates_content)
+        capture_dates = {
+            "href": f"./{CAPTURE_DATES_FILE_NAME}",
+            "title": "Capture dates",
+            "type": "application/gzip",
+            "roles": ["metadata"],
+            "file:checksum": file_checksum,
```

Comment:
are we checksumming the compressed content or the content?

Reply:
We checksum the compressed file.

Reply:
Hmm this is a interesting one, as given the uncompressed file it would be difficult to know how to compress it again to get the same file.

Is `gzip -9 capture-dates.geojson` the same as `gzipSync(...)` vs `gzip.compress(...)` I am assuming not so it would have to be compressed in the same way to get the same hash

Reply:
We should checksum whatever is stored in S3, without transforming it first. Otherwise the checksum isn't very useful.

Reply:
Decision made not to gzip the capture dates file :+1: